Deprecations for Volto 19 (#6848)
plone/volto: docs/source/_inc/_branch-policy.md

The Volto team enforces the following branch policy when developers contribute to its core.

> Releases of general packages (`@plone/scripts`, and so on) are cut from the `main` branch.

Reviewer: ```suggestion
Releases of general packages (`@plone/scripts` and other packages) are cut from the `main` branch.
```
